docs/configuration/adding_your_own.md

```diff
@@ -0,0 +1,86 @@
+
+## Cluster Environment
+By default, pipeline uses the `local` Nextflow executor - in other words, all jobs are run in the login session. If you're using a simple server, this may be fine. If you're using a compute cluster, this is bad as all jobs will run on the head node.
+
+To specify your cluster environment, add the following line to your config file:
+
+```nextflow
+process.executor = 'YOUR_SYSTEM_TYPE'
+```
+
+Many different cluster types are supported by Nextflow. For more information, please see the [Nextflow documentation](https://www.nextflow.io/docs/latest/executor.html).
+
+Note that you may need to specify cluster options, such as a project or queue. To do so, use the `clusterOptions` config option:
+
+```nextflow
+process {
+  executor = 'SLURM'
+  clusterOptions = '-A myproject'
+}
+```
+
+
+## Software Requirements
+To run the pipeline, several software packages are required. How you satisfy these requirements is essentially up to you and depends on your system. If possible, we _highly_ recommend using either Docker or Singularity.
+
+Please see the [`installation documentation`](../installation.md) for how to run using the below as a one-off. These instructions are about configuring a config file for repeated use.
+
+### Docker
+Docker is a great way to run nf-core/rnafusion, as it manages all software installations and allows the pipeline to be run in an identical software environment across a range of systems.
+
+Nextflow has [excellent integration](https://www.nextflow.io/docs/latest/docker.html) with Docker, and beyond installing the two tools, not much else is required - nextflow will automatically fetch the [nfcore/rnafusion](https://hub.docker.com/r/nfcore/rnafusion/) image that we have created and is hosted at dockerhub at run time.
+
+To add docker support to your own config file, add the following:
+
+```nextflow
+docker.enabled = true
+process.container = "nfcore/rnafusion"
+```
+
+Note that the dockerhub organisation name annoyingly can't have a hyphen, so is `nfcore` and not `nf-core`.
+
+
+### Singularity image
+Many HPC environments are not able to run Docker due to security issues.
+[Singularity](http://singularity.lbl.gov/) is a tool designed to run on such HPC systems which is very similar to Docker.
+
+To specify singularity usage in your pipeline config file, add the following:
+
+```nextflow
+singularity.enabled = true
+process.container = "shub://nf-core/rnafusion"
+```
+
+If you intend to run the pipeline offline, nextflow will not be able to automatically download the singularity image for you.
+Instead, you'll have to do this yourself manually first, transfer the image file and then point to that.
+
+First, pull the image file where you have an internet connection:
+
+```bash
+singularity pull --name nf-core-rnafusion.simg shub://nf-core/rnafusion
```

Comment:
We removed `shub://` from nf-core, as we are only pulling from docker hub now - just replace all occurences of `shub://` with `docker://`, that should be fine

Reply:
Done